main/src/ca/uwaterloo/flix/language/phase/Weeder2.scala

```diff
@@ -1095,7 +1094,7 @@ object Weeder2 {
 
     private def visitArguments(tree: Tree)(implicit sctx: SharedContext): Validation[List[Expr], CompilationMessage] = {
       mapN(
-        traverse(pickAll(TreeKind.Argument, tree))(pickExpr),
+        traverse(pickAll(TreeKind.Argument, tree))(pickExpr(_)),
```

Comment:
Scala requires this?

Reply:
Yes, it's a type error otherwise (with the default parameter it apparently needs explicit application)